acceptance/unstable/.kitchen.yml

```diff
@@ -38,3 +38,16 @@ suites:
       product_version: 0.10.0+20151209030614
       channel: unstable
     run_list:
+  - name: chef-unstable-install-latest
+    provisioner:
+      product_name: chef
+      product_version: latest
```

Comment:
Rather than adding a new suite (which multiplies the number of test cases) let's switch chefdk to use `:latest`.